test/controllers/returns/ReturnsCheckYourAnswersControllerSpec.scala

```diff
@@ -82,7 +82,7 @@ class ReturnsCheckYourAnswersControllerSpec extends PlaySpec with SummaryListFlu
   private val mockTaxReturnHelper = mock[TaxReturnHelper]
   private val appConfig = mock[FrontendAppConfig]
   private val navigator = mock[ReturnsJourneyNavigator]
-  private val keyString = "a-key"
+  private val keyString = s"${LocalDate.now()}-${LocalDate.now()}"
```

Comment:
Is it possible to have literal dates here rather than now()?

Reply:
Ok, I see. Would something like "2000-01-02-2001-03-04" work?

Reply:
But either way, something for later I think

Reply:
yes that would work too, assuming the useranswers also lined up